addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/create.html

```diff
@@ -136,6 +157,11 @@ <H3>reportData</H3>
 	<p>Automation jobs can make additional data available to the
 		reports, in many cases even if the automation framework was not used.
 		See the help pages for the relevant jobs for more details.</p>
+
+	<H3>stats</H3>
+	A Map of the global statistics. The site based statistics are available
```

Comment:
Would "table" be better than "map" from an end user perspective?

Reply:
The audience is the template creators, and they need to know what type of object it is. This info is not relevant to report users :)